### PR TITLE
[WIP][WhistleOut] Remove wildcard and simplify rule

### PR DESCRIPTION
**Pull request by Bisaloo**

### CODE REVIEW

File: src/chrome/content/rules/WhistleOut.xml [hunk 1]
```diff
@@ -1,18 +1,29 @@
...
+	<target host="techradar.whistleout.com" />
+	<target host="telegraph.whistleout.com" />
+	<target host="test.whistleout.com" />
+	<target host="trustedreviews.whistleout.com" />
+
 	<target host="whistleout.com.au" />
 	<target host="www.whistleout.com.au" />
+
 	<target host="whistleout.co.uk" />
 	<target host="www.whistleout.co.uk" />
```
Comment (koops76) [inline src/chrome/content/rules/WhistleOut.xml]: Add `<securecookie host=".+" name=".+" />`.

Comment (Bisaloo) [inline src/chrome/content/rules/WhistleOut.xml]: I'm not securing cookies without extensive tests. That's one of the major causes of issues.

Comment (koops76) [inline src/chrome/content/rules/WhistleOut.xml]: @Bisaloo Okay.

**Comment by J0WI** [comment]
@koops76 since you already approved it, what prevents you from merging this?

**Comment by koops76** [comment]
@J0WI no write access.

**Comment by J0WI** [comment]
@koops76 Just ping me if something is ready to commit.

**Pull request merged**